OpenKh.Patcher/PatcherProcessor.cs

```diff
@@ -279,5 +284,79 @@ private static void PatchAreaDataScript(Context context, List<AssetFile> sources
 
             Kh2.Ard.AreaDataScript.Write(stream.SetPosition(0), scripts.Values);
         }
+
+        private static void PatchListReplace(Context context, List<AssetFile> sources, Stream stream)
+        {
+            foreach (var source in sources)
+            {
+                var deserializer = new DeserializerBuilder().IgnoreUnmatchedProperties().Build();
+                Dictionary<string, byte> characterMap = new Dictionary<string, byte>() {
+                            { "Sora", 1 }, { "Donald", 2 }, { "Goofy", 3 },  { "Mickey", 4 },  { "Auron", 5 }, { "PingMulan",6 }, { "Aladdin", 7 },  { "Sparrow", 8 }, { "Beast", 9 },  { "Jack", 10 },  { "Simba", 11 }, { "Tron", 12 }, { "Riku", 13 }, { "Roxas", 14}
+                        };
+                    string sourceText = File.ReadAllText(context.GetSourceModAssetPath(source.Name));
+                    if (source.Name.Contains("enc"))
```

Comment:
I know that encoding the file as base64 is to obfuscate files to not cheat on randomizers, but I prefer to not couple this functionality with the patcher functionality. So I'd say to remove the next 5 lines of code.
I was thinking on a way to randomize values on patching time rather than modfying the mod files. But it would be out of scope of this PR anyway.

Reply:
But then you need to build a randomiser into the mod launcher? Personally instead of using enc in the file name I would use an tag in the yaml just to prevent issues with other mod creators using enc as a file name.